docs/output.md

```diff
@@ -5,16 +5,26 @@ This page describes the output files and report produced by the pipeline. The di
 
 ## Pipeline Overview
 
-This pipeline takes in metagenomic reads, performs QC, assembly, and subsequent mapping of reads back to the resulting assemblies. Steps downstream of assembly and mapping are identical for processing data from both sequencing types.
+This pipeline takes in metagenomic samples produced from either Illumina or Nanopore technologies and performs common QC, processing, and profiling steps. **Important note**: This pipeline separately processes Illumina or Nanopore metagenomes and does not perform hybrid assembly, polishing of Nanopore assemblies with Illumina reads, or scaffolding with Nanopore reads.
+
+This pipeline is intended to automate the common QC and processing steps to ensure seamless and reproducible processing of metagenomes for downstream analysis. Therefore the end goal of the pipeline regardless of which sequencing technology used is to produce QC stats of the reads, generate assemblies (including polishing for Nanopore), map the original reads back to the assemblies and get mapping statistics, and summarize the composition of the metagenomes.
+
+Since Illumina or Nanopore metagenomes have to be processed separately, you can find the corresponding results files produced from each step of the pipeline in a folder with the name of the tool. For example, QCed Illumina reads will be in a subfolder of your output results directory called `fastp/`.
```

Comment:
no depending on the run it will just be the tool name without appended with the workflow name. So it will be for  example the result outfolder named `illumina_results` and then `fastp` within that